contrib/inventory/consul_io.py

```diff
@@ -133,7 +133,11 @@
 import re
 import argparse
 import sys
-import ConfigParser
+
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
@pilou- 

First: As this will only make 2.4 it appears that the case test isn't needed and `import configparser` should be fine for the ansible 2.4 supported versions.

I do want to point out that the design decision to have inventory scripts external to the controller makes it problematic to assume that `from ansible.module_utils import six` is a possibility.

There are several use cases where the generated file may be provided and generated outside of the controller context like Vagrant, or if resiliency is required against the external service registry. 

I admit I am being over sensitive, just because I have to have a playbook to make sure that my controllers modules work, but as only 6 out of 51 contrib/inventory scripts have `from ansible.module_utils` includes I am unwilling to add that un-tracked dependency.
Example related to resilience and loose coupling.

https://medium.com/@Pinterest_Engineering/zookeeper-resilience-at-pinterest-adfd8acf2a6b

I will update and test as time allows.